sockeye/checkpoint_decoder.py

```diff
@@ -87,7 +89,13 @@ def decode_and_evaluate(self, checkpoint: int) -> Dict[str, float]:
         :param checkpoint: Checkpoint to load parameters from.
         :return: Mapping of metric names to scores.
         """
-        translator = sockeye.inference.Translator(self.context, 'linear',
+        ensemble_mode = 'linear'
+        bucket_source_width = 10
+        bucket_target_width = 10
```

Comment:
Eh no, just forgot to change to latest default values

Reply:
Actually, I'd like to leave at 10:10 since these processes are started many times for just a couple of sentences (subsample of validation data). Its probably not worth it to set it to 10:2 as this causes allocation of a lot more graphs

Reply:
makes sense. This will also probably reduce the mem size, no?

Reply:
yes, but allocating more graphs has a surprisingly small effect on memory usage. Memory sharing is not too bad these days in MXNet :)